.github/workflows/release.yml

```
@@ -0,0 +1,177 @@
+        uses: actions/download-artifact@v3
+        with:
+          path: dist/
+
+      - name: Flatten artifacts
+        run: |
+          mkdir -p release/
+          find dist/ -name "tilokit-*" -type f -exec mv {} release/ \;
+          ls -la release/
+
+      - name: Generate checksums
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2035:info:2:11: Use ./*glob* or -- *glob* so names with dashes won't become options

(shellcheck)